Makefile

```diff
@@ -32,7 +32,7 @@ clean:
 	rm ./bin -rf
 
 distclean: clean
-	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
+	test -d vendor && find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
```

Comment:
Ajajaj, I just verified it, and this revision is not enough, I need to append `|| true` at the end.

Reply:
I just realized now, that  this is just a left-over from the time when we usede govendor instead of glide. `rm -rf vendor/` is good enough here.

Reply:
OK